scripts/set-alias-page.py

```diff
@@ -100,10 +100,13 @@ def get_alias_page(file):
         return ""
     with open(file) as f:
         lines = f.readlines()
-    for line in lines:
-        if re.search(r"^`tldr ", line):
-            return re.search("`tldr (.+)`", line).group(1)
-    return ""
+    return next(
```

Comment:
I think all the other changes here are sensible, but just have some questions about this one!

I don't see it making a real-world performance difference, but this PR moves from searching until a match is found to searching all lines. So it does an extra iteration per page over the previous implementation. While it's more pythonic, I don't think it's any more readable/maintainable anyway.

Perhaps this one can be changed back? Alternatively, do you have any thoughts on this implementation?

```py
if not os.path.isfile(file):
    return ""
with open(file) as f:
    for line in f:
        if match := re.search(r"^`tldr (.+)`", line):
            return match[1]
return ""
```

* We can just iterate while reading the file rather than read it all upfront.
* There's no need to call `re.search` twice, we can just check if the match is Truthy.

Reply:
Interesting, first time knowing `next`. After viewing the examples from [geeksforgeeks](https://www.geeksforgeeks.org/python-next-method/), I also think utilizing `next` in this code is less ideal for readability.
@RemoteRabbit Also, please run the [test script](https://github.com/tldr-pages/tldr/blob/main/scripts/test.sh) to pass the coding style check.

Reply:
> Perhaps this one can be changed back? Alternatively, do you have any thoughts on this implementation?
> 
> ```python
> if not os.path.isfile(file):
>     return ""
> with open(file) as f:
>     for line in f:
>         if match := re.search(r"^`tldr (.+)`", line):
>             return match[1]
> return ""
> ```

I agree with this setup, will update and push that change

Reply:
updated